crates/forge/src/running.rs

```diff
@@ -166,3 +170,41 @@ pub(crate) fn run_from_test_case(
         Err(err) => Err(err.into()),
     }
 }
+
+fn get_fork_state_reader(
```

Comment:
I would instead make the fork_config non-optional, and return ForkStateReader instead of an Option

Reply:
mostly it's optional, because ExtendedStateReader wants optional:
```
pub struct ExtendedStateReader {
    pub dict_state_reader: DictStateReader,
    pub fork_state_reader: Option<ForkStateReader>,
}

```